src/main/webapp/Portal.html

```diff
@@ -8,6 +8,11 @@
     />
     <meta name="author" content="Sage Bionetworks, info@sagebase.org" />
     <meta http-equiv="Content-Type" content="text/html; charset=utf-8" />
+
+    <meta
+      http-equiv="Content-Security-Policy"
+      content="default-src 'self' data: 'unsafe-inline' 'unsafe-eval' *.synapse.org *.sagebase.org *.googleapis.com *.gstatic.com *.googletagmanager.com cdn.statuspage.io s3.amazonaws.com www.google-analytics.com kh896k90gyvg.statuspage.io 127.0.0.1:9876"
```

Comment:
This seems like an appropriate list. Just wondering how you came up with it?

Reply:
Is it possible to embed external images in Wiki markdown? Wonder if we need to add `img-src *`. [MDN](https://developer.mozilla.org/en-US/docs/Web/HTTP/Headers/Content-Security-Policy/img-src).

Reply:
Would this also affect the contents of an `iframe`d HTML preview? I am not currently sure what functionality remains for that feature

Reply:
Nice catch, I'll add it